### DISCUSSION TITLE
3.10.0

### DISCUSSION THREAD

**Discussion by Davidy22** [Guake/guake#2176] [Announcements]
Featuring customisable search engines, the ability to properly open new tabs when invoked as a default terminal, a pin button and various other fixes and features, Guake 3.10 is now out.

## What's Changed
* Fix 0 timestamp #1857 #1950 by @c31io in https://github.com/Guake/guake/pull/2102
* search engine can be set to preference by @PhantomLel in https://github.com/Guake/guake/pull/2125
* Added custom search engine input #2088 by @PhantomLel in https://github.com/Guake/guake/pull/2134
* Include names of processes in PromptQuitDialog by @hackedd in https://github.com/Guake/guake/pull/2135
* Add new hotkey to open new tab in same directory by @hackedd in https://github.com/Guake/guake/pull/2136
* Add pin button to temporarily prevent hiding when losing focus by @hackedd in https://github.com/Guake/guake/pull/2138
* Add pre-commit config and apply all fixes by @hackedd in https://github.com/Guake/guake/pull/2139
* Add a new hotkey to select all text in the current terminal by @hackedd in https://github.com/Guake/guake/pull/2145
* Run -e in new tabs generated by split vertical/horizontal by @Davidy22 in https://github.com/Guake/guake/pull/2172
* Add global hotkey instructions to docs by @Davidy22 in https://github.com/Guake/guake/pull/2173
* Open a new guake tab if guake is called with `guake [PATH]` and no other flags. by @Davidy22 in https://github.com/Guake/guake/pull/2174
* Save font size when changing zoom level with keyboard shortcuts by @Davidy22 in https://github.com/Guake/guake/pull/2175

## New Contributors
* @c31io made their first contribution in https://github.com/Guake/guake/pull/2102
* @PhantomLel made their first contribution in https://github.com/Guake/guake/pull/2125

**Full Changelog**: https://github.com/Guake/guake/compare/3.9.0...3.10

<hr /><em>This discussion was created from the release <a href='https://github.com/Guake/guake/releases/tag/3.10'>3.10.0</a>.</em>

**Comment by baptistedonaux** [comment]
I love Guake, so firstly, thank you for your work!

I use it on Fedora 38 and I can see that latest version available in repository is `3.7.0`. Is it possible to release it?

**Reply by Davidy22** [reply]
After Guake is released, it's up to the individual distro package maintainers to ship the new versions to their respective repositories. The [record shows a couple of different people](https://bodhi.fedoraproject.org/updates/?packages=guake) who have handled previous Guake versions, but there does seem to have been a gap since the last release. You can try to contact one of the previous packagers, or you can get a contribution to Fedora under your name and be the one to update Fedora to 3.10.
